example/tests/elementActions.test.js

```diff
@@ -233,6 +233,19 @@ describe("Element Actions", () => {
         expect(await formElement.getAttributeValue(attributeName)).toEqual("/authenticate");
     });
 
+    it("should set element's text value", async () => {
```

Comment:
I suggest changing word "set" to "enter" or "type", as there is another approach in puppeteer to enter value, by just setting it, so we should not confuse this with that.